.npmrc

```diff
@@ -0,0 +1,2 @@
+link-workspace-packages = deep
```

Comment:
What is the purpose of this .npmrc file? Did you experience something weird without it?

Reply:
I missed this one. I think pnpm generated it automatically for some reason. Removing it